src/test/java/integration/helper/TestHelper.java

```diff
@@ -160,7 +166,7 @@ public Client createClient(Counterparty counterparty) throws Exception {
         return clientService.saveEntity(client, client.getCounterparty().getUser());
```

Comment:
This method cannot be removed as it is used in setUp() in ClientControllerIT, where it does not have sender/recipient naming context.